app.cfg.dist

```diff
@@ -0,0 +1,14 @@
+[oauth2]
+    [[clients]]
+        [[[client-name]]]
```

Comment:
This feels wrong, but not sure how to allow for nesting.

Reply:
ah, `configobj`, it's the simplest solution 
Other people use convention:
`oauth2_clients_client_name: ...`